From: Alexander Graf <agraf@suse.de>
Subject: [PATCH] MMC: Exynos: Add module alias for dw mmc
Patch-Mainline: Not yet

We are missing a module link in the dw mmc exynos driver
back to its module. This confuses the openSUSE mkinitrd
logic, so that we can not find the module as the driver for
the root device on a running system.

Signed-off-by: Alexander Graf <agraf@suse.de>

Index: linux-3.10-master/drivers/mmc/host/dw_mmc-exynos.c
===================================================================
--- linux-3.10-master.orig/drivers/mmc/host/dw_mmc-exynos.c
+++ linux-3.10-master/drivers/mmc/host/dw_mmc-exynos.c
@@ -194,6 +194,7 @@ static struct platform_driver dw_mci_exy
 	.remove		= dw_mci_pltfm_remove,
 	.driver		= {
 		.name		= "dwmmc_exynos",
+		.owner		= THIS_MODULE,
 		.of_match_table	= dw_mci_exynos_match,
 		.pm		= &dw_mci_exynos_pmops,
 	},
